plugins/mumble_plugin_main.h

```diff
@@ -29,6 +35,8 @@ static bool is64Bit;
 static procid_t pPid;
 static procptr_t pModule;
 
+#define POINTER_SIZE (is64Bit ? 8 : 4)
```

Comment:
Wouldn't it be easier and more robust to use the `sizeof` operator? Something like
```
#define POINTER_SIZE` sizeof(int*)
```
should do the trick, shouldn't it?

Reply:
It wouldn't work in case the target process' architecture is different from Mumble's one.

Reply:
Are you sure about that? How can the compiler create a usable executable for that architecture, if it uses the wrong pointer-size for it?
I'd assume that the compiler always spits out the correct size here, as I don't see how cross-compilation could work otherwise...

Reply:
`POINTER_SIZE` only determines the amount of bytes we read from the target process:
https://github.com/mumble-voip/mumble/blob/72e1f47b7b13d2ffd2a98cb2fd79c10674376698/plugins/mumble_plugin_main.h#L55-L63

Reply:
Ah okay and the problem is that even though the plugin is compiled for 64bit, you kigjt have to read from a 32bit application? Thus this has to be determined at runtime... Correct?

Reply:
Correct!

The opposite is also possible: 32 bit plugin reading from a 64 bit process.

Reply:
Okay then. I think the name of the macros somehow implies that it was a Co stant though. I think calling it `GET_POINTER_SIZE` would be more descriptive...

Reply:
Right.